modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/api/api.ts

```diff
@@ -3,6 +3,38 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {API as objectAPI} from '@liferay/object-js-components-web';
+
+export const {save: saveObject} = objectAPI;
+
+const UNEXPECTED_ERROR_MESSAGE = Liferay.Language.get(
+	'an-unexpected-error-occurred'
+);
+
+export async function postFormData(
+	formData: FormData,
+	url: string
+): Promise<{errorMessage: string | null; success: boolean}> {
```

Comment:
Can we `type` this `{errorMessage: string | undefined; success: boolean}`?

With this change we can change the return of success

**From**
```ts
{
	errorMessage: null,
	success: true,
}
```

**to**
```ts
{success: true}
```